cognite/neat/rules/models/_rules/_types/_field.py

```diff
@@ -247,12 +248,14 @@ def _semantic_value_type_before_validator(value: Any) -> Any:
 ]
 
 CdfValueType = Annotated[
-    DMSValueType | ViewEntity,
+    DMSValueType | ViewPropEntity,
```

Comment:
is this correct? ValueType can be either another View or simple values, I am not sure how it can be Value Property?

Reply:
See PR description.